components/pagination/index.php

```diff
@@ -148,6 +178,32 @@
     }
     // End of LengthAwarePaginator example
 
+    // Create ViewFactory instance -- see the view component for more info
+    $events = new Dispatcher(new Container);
+
+    $pathsToTemplates = [__DIR__ . '/templates'];
+
+    $filesystem = new Filesystem;
+
+    $viewResolver = new EngineResolver;
+    $viewResolver->register('php', function () {
+        return new PhpEngine;
+    });
+
+    $viewFinder = new FileViewFinder($filesystem, $pathsToTemplates);
+
+    $viewFactory = new Factory($viewResolver, $viewFinder, $events);
+    // End of create ViewFactory instance
+
+    // Set the view factory resolver
```

Comment:
I wondered the same thing. I'm sort of leaning toward yes, since it's a prep step that someone would put in a global/boot step, so it makes sense to pull it out. Thanks for asking!

Reply:
Done. Sorry for titling the PR `Refactoring ViewFactory`, couldn't come up with a better one!